dcos/subcommand.py

```diff
@@ -61,6 +66,72 @@ def get_package_commands(package_name):
     return executables
 
 
+def default_subcommands():
+    """List the default dcos cli subcommands
+
+    :returns: list of all the default dcos cli subcommands
+    :rtype: [str]
+    """
+
+    return ["config", "help", "marathon", "node", "package", "service", "task"]
+
+
+def default_modules():
+    """Dict of the default dcos cli subcommands and their main methods
+
+    :returns: default subcommand -> main method
+    :rtype: {}
+    """
+
+    # avoid circular imports
+    from dcoscli.config import main as config_main
+    from dcoscli.help import main as help_main
+    from dcoscli.marathon import main as marathon_main
+    from dcoscli.node import main as node_main
+    from dcoscli.package import main as package_main
+    from dcoscli.service import main as service_main
+    from dcoscli.task import main as task_main
+
+    return {'config': config_main,
+            'help': help_main,
+            'marathon': marathon_main,
+            'node': node_main,
+            'package': package_main,
+            'service': service_main,
+            'task': task_main
+            }
```

Comment:
This is duplication that is going to be difficult to track down when we add more subcommands.

```
_default_modules = {
  'config': from dcoscli.config import main,
  'help': from dcoscli.help import main,
  ...
}

def default_subcommands():
   return _default_modules.keys()
```

You can remove `def default_modules()`. It is not used outside the module. Also, we need to move this code to the `dcoscli` package. The `dcoscli` package depends on `dcos`, not the other way around.